### DISCUSSION TITLE
What the F are we building

### DISCUSSION THREAD

**Discussion by inquisitev** [entropic-entertainment/endless-runner#2] [Ideas]
GO!

**Comment by KainDevs** [comment]
A simple fantasy endless runner where a torso-only, floating, hunchback wizard is trying to escape a running fat dragon with small wings.

**Reply by inquisitev** [reply]
So tasks to make that include
- [ ] Create character model
- [ ] Create dragon model
- [ ] Create obstacle models
- [ ] Create reward model objects
- [ ] Move player model at constant speed
- [ ] Spawn obstacles in players path
- [ ] Bind a key button to move left or move right
- [ ] End game if player collides with obstacles
- [ ] Add earned Reward indication
- [ ] Save earned rewards and display in indication

**Reply by inquisitev** [reply]
but that doesnt cover sound and other stuff
